.github/workflows/build.yaml

```diff
@@ -63,27 +67,23 @@ jobs:
         run: >-
           touch .env.development;
           touch .env;
-          TAG=latest docker-compose pull web || true
+          docker pull ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:latest
```

Comment:
On this step in the workflow I was getting an error for pulling the image tag from the registry. I assumed it was because it hadn't been built or pushed to the registry yet. Which is how I came to the assumption the code prior was trying to pull tag latest. When I tried it out it worked.